aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -16,6 +16,8 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 [!INCLUDE [consider RP](~/includes/razor.md)]
```

Comment:
Update this include
`
This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point. See [choose UI ...](_) which compares ... `